unified-runtime/source/adapters/offload/ur_interface_loader.cpp

```diff
@@ -398,6 +398,7 @@ UR_DLLEXPORT ur_result_t UR_APICALL urGetProgramExpProcAddrTable(
   pDdiTable->pfnBuildExp = urProgramBuildExp;
   pDdiTable->pfnCompileExp = nullptr;
   pDdiTable->pfnLinkExp = nullptr;
+  pDdiTable->pfnDynamicLinkExp = nullptr;
```

Comment:
Should the offload adapter not provide a value for `UR_DEVICE_INFO_DYNAMIC_LINK_SUPPORT_EXP` as well?

Reply:
Good eye! I totally missed this. Most likely because I grepped for `UR_DEVICE_INFO_MULTI_DEVICE_COMPILE_SUPPORT_EXP` which isn't covered by the offload adapter currently. Neither is `UR_DEVICE_INFO_USM_P2P_SUPPORT_EXP` I've noticed.